xml/kernel_modules.xml

```diff
@@ -0,0 +1,179 @@
+    currently loaded. The output of the command may look as follows:
+  </para>
+  <screen>$ lsmod
+Module                  Size  Used by
+snd_usb_audio         188416  2 
+snd_usbmidi_lib        36864  1 snd_usb_audio
+hid_plantronics        16384  0 
+snd_rawmidi            36864  1 snd_usbmidi_lib
+snd_seq_device         16384  1 snd_rawmidi
+fuse                  106496  3 
+nfsv3                  45056  1 
+nfs_acl                16384  1 nfsv3</screen>
+  <para>
+    The output is divided into three columns. The <literal>Module</literal> column lists the names
+    of the loaded modules, while the <literal>Size</literal> column displays the size of each
+    module. The <literal>Used by</literal> column shows the number of referring
+    modules and their names. Note that this list may be incomplete.
+  </para>
+  <para>
+    To view detailed information about a specific kernel module, use the
+    <command>modinfo <replaceable>MODULE_NAME</replaceable></command> command, where
+    <replaceable>MODULE_NAME</replaceable> is the name of the desired kernel
+    module. Note that the <command>modinfo</command> binary resides in the
+    <filename>/sbin</filename> directory that is not in the user's PATH
+    environment variable. This means that you must specify the full path to the
+    binary when running <command>modinfo</command> command as a regular user:
+  </para>
+  <screen>$ /sbin/modinfo kvm
+filename:       /lib/modules/4.4.57-18.3-default/kernel/arch/x86/kvm/kvm.ko
+license:        GPL
+author:         Qumranet
+srcversion:     BDFD8098BEEA517CB75959B
+depends:        irqbypass
+intree:         Y
+vermagic:       4.4.57-18.3-default SMP mod_unload modversions 
+signer:         openSUSE Secure Boot Signkey
+sig_key:        03:32:FA:9C:BF:0D:88:BF:21:92:4B:0D:E8:2A:09:A5:4D:5D:EF:C8
+sig_hashalgo:   sha256
+parm:           ignore_msrs:bool
+parm:           min_timer_period_us:uint
+parm:           kvmclock_periodic_sync:bool
+parm:           tsc_tolerance_ppm:uint
+parm:           lapic_timer_advance_ns:uint
+parm:           halt_poll_ns:uint
+parm:           halt_poll_ns_grow:int
+parm:           halt_poll_ns_shrink:int</screen>
+  </sect1>
+  <sect1 xml:id="sec.mod.modprobe">
+    <title>Adding and Removing Kernel Modules</title>
+    <para>
+      While it is possible to use the <systemitem>insmod</systemitem> and
+      <systemitem>rmmod</systemitem> to add and remove kernel modules, it is recommended to use the
+      <systemitem>modprobe</systemitem> tool instead. <systemitem>modprobe</systemitem> offers several
+      important advantages, including automatic dependency resolution and
+      blacklisting.
+    </para>
+    <para>
+      When used without any parameters, the <systemitem>modprobe</systemitem>
+      command installs a specified kernel. <systemitem>modprobe</systemitem> must be run with root privileges:
+    </para>
+    <screen>sudo modprobe acpi</screen>
+    <para>
```

Comment:
Prompt is missing. "Command-sreens" without prompts are no longer allowed ;-((

&lt;screen>&prompt.user;sudo modprobe acpi&lt;/screen>

Also applies to other screens below